andela/travel_tool_front
# #161866839 Change submit comment button label

#### What does this PR do?
This PR corrects the typographical error of the button label to post a comment

#### Description of Task to be completed?
- changes the label of the button from 'Posts' to 'Post'
 
#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory -` cd travel_tool_front`
- Pull and checkout into this branch - `ch-change-comment-button-label-161866839`
- Install project dependencies using `yarn install`
- Clone the [Travela backend](https://github.com/andela/travel_tool_back)
- Run database migrations using `yarn run db:rollmigrate`
- Check `.env.example` and set up your `.env` appropriately
- Start the Server and Frontend with `make start` and log into the application
- Create a new travel request
- Click on the created request Id to view the request details
- The button to post a comment should be labelled 'Post' not 'Posts'

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#161866839](https://www.pivotaltracker.com/story/show/161866839)

#### Screenshots (if appropriate)
<img width="980" alt="screen shot 2018-11-13 at 2 41 30 pm" src="https://user-images.githubusercontent.com/32720508/48417587-648c0580-e753-11e8-92e0-1ed011dc5260.png">


#### Questions:
N/A
